config/config.go

```diff
@@ -1087,9 +1105,11 @@ func getSubnetConfigsFromDir(v *viper.Viper, subnetIDs []ids.ID) (map[ids.ID]sub
 			return nil, fmt.Errorf("%w: %w", errUnmarshalling, err)
 		}
 
-		if config.ConsensusParameters.Alpha != nil {
-			config.ConsensusParameters.AlphaPreference = *config.ConsensusParameters.Alpha
-			config.ConsensusParameters.AlphaConfidence = config.ConsensusParameters.AlphaPreference
+		if config.ConsensusParameters.SimplexParams != nil {
+			setSimplexDefaults(&config.ConsensusParameters, v)
+		} else if config.ConsensusParameters.SnowballParams.Alpha != nil {
```

Comment:
Why `config.ConsensusParameters.SnowballParams.Alpha` and not `config.ConsensusParameters.SnowballParams` ?